src/loop.jl

```diff
@@ -106,24 +106,17 @@ function interval_loop(
                     getfield(case, Symbol(p))[interval_start:interval_end, 2:end]
                 )
             end
-            for g in keys(sets.profile_resources_num_rep)
+            for g in values(sets.profile_resources_num_rep)
```

Comment:
I think we can just use `for g in case.profile_resources` instead. If so, I believe `profile_resources_num_rep` can be removed from `sets`.